e2e-tests/tfm/cypress/e2e/pages/facilityPage.js

```diff
@@ -16,6 +16,9 @@ const facilityPage = {
   facilityBankIssueNoticeReceived: () => cy.get('[data-cy="facility-bank-issue-notice-received"]'),
   facilityCoverStartDate: () => cy.get('[data-cy="facility-cover-start-date"]'),
   facilityCoverEndDate: () => cy.get('[data-cy="facility-cover-end-date"]'),
+  facilityIsUsingFacilityEndDate: () => cy.get('[data-cy="facility-is-using-facility-end-date"]'),
```

Comment:
maybe this const and data-cy could be renamed to `facilityIsUsingEndDate` and `'[data-cy="facility-is-using-end-date"]'`?

Reply:
see before (keeping convention)